dse/install-artifacts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+DSE_VERSION=6.8.16
+mvn dependency:get -Dartifact=com.datastax:dse-db-all:${DSE_VERSION} -o && mvn dependency:get -Dartifact=com.datastax:dse-commons:${DSE_VERSION} -o && mvn dependency:get -Dartifact=io.netty:netty-all:4.1.25.7.dse
```

Comment:
could we maybe redirect error output to dev null or something like that? What I see locally is that it will try to get this and if it is not OK, it will download image etc. What I kind of do not like to see is that error but that is just a personal preference ....